### PR TITLE
Feature/tank level

### PR DESCRIPTION
**Pull request by Hunter27**

### CODE REVIEW

File: WaterLog_Backend/Procedures.cs [hunk 1]
```diff
@@ -1065,6 +1066,31 @@ public List<MonitorHeat> getMonitorsFaultLevels()
             return daily;
 
         }
+
+        public async Task<List<TankObject>> getObjects()
+        {
+            var allGrouped  = await _db
+                            .TankReadings.Where(a => a.TimeStamp.Month == DateTime.Now.Month && a.TimeStamp.Year == DateTime.Now.Year )
```
Comment (FredJEhlers) [inline WaterLog_Backend/Procedures.cs]: This line is too long here is a suggestion 
```
var allGrouped = await _db
	.TankReadings
        .Where(a => a.TimeStamp.Month == DateTime.Now.Month && a.TimeStamp.Year == DateTime.Now.Year )
	.GroupBy(b => b.TankMonitorsId)
	.ToListAsync();
```

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: WaterLog_Backend/Procedures.cs (hunk 1 above)
Comment (Hunter27) [reply WaterLog_Backend/Procedures.cs]: No anonymous variable, function programming like lambda function in a sense.
ill indent it better?

Comment (Hunter27) [reply WaterLog_Backend/Procedures.cs]: ohhh sorry misunderstood you

Comment (Hunter27) [reply WaterLog_Backend/Procedures.cs]: changed it

**Review by FredJEhlers — APPROVED** [review]

**Review by ash-rabbit — APPROVED** [review]

**Pull request merged**